### PR TITLE
Add a setting for allowing an indeterminate progress bar

### LINKED ISSUE #80: Showing negative numbers as progress bars doesn't work
Showing negative numbers as progress bars does the cylon eyes thing - very cool, but probably not what was expected.

### PR DESCRIPTION
**Pull request by AustinShalit**
closes #80 

ezgif-3-80740cfe4e

### CODE REVIEW

**Comment by SamCarlberg** [comment]
We should probably also make the default range (-1, 1) for consistency with the slider

File: src/main/java/edu/wpi/first/shuffleboard/widget/ProgressBar.java [hunk 1]
```diff
@@ -19,12 +22,13 @@
 
   final DoubleProperty minValue = new SimpleDoubleProperty(this, "Min Value", 0);
   final DoubleProperty maxValue = new SimpleDoubleProperty(this, "Max Value", 1);
+  final BooleanProperty allowIndeterminateValue = new SimpleBooleanProperty(this, "Allow Indeterminate", false);
```
Comment (AustinShalit) [inline src/main/java/edu/wpi/first/shuffleboard/widget/ProgressBar.java]: I do not understand what change you need me to make.

Comment (AustinShalit) [inline src/main/java/edu/wpi/first/shuffleboard/widget/ProgressBar.java]: #80 claims that the default value should be false.  @bradamiller thoughts?

Comment (nightpool) [inline src/main/java/edu/wpi/first/shuffleboard/widget/ProgressBar.java]: allowIndeterminateValue -> allowIndeterminate

Comment (SamCarlberg) [inline src/main/java/edu/wpi/first/shuffleboard/widget/ProgressBar.java]: I was referring to the the string

**Comment by nightpool** [comment]
To reiterate, having this as a setting doesn't make sense. it should just be the default behavior to constrain the value to a range.

**Comment by AustinShalit** [comment]
@nightpool:
>This should at the very least be the default behavior.

What do you think should be the default behavior?

@SamCarlberg:
>We should probably also make the default range (-1, 1) for consistency with the slider

Good idea.  I will open a PR.

**Comment by nightpool** [comment]
closed in favor of #88

**Pull request closed without merging**